common/persistence/sql/sqlplugin/interfaces.go

```diff
@@ -31,6 +31,8 @@ type (
 	// Plugin defines the interface for any SQL database that needs to implement
 	Plugin interface {
 		CreateDB(dbKind DbKind, cfg *config.SQL, r resolver.ServiceResolver, l log.Logger, mh metrics.Handler) (GenericDB, error)
+
+		GetVisibilityQueryConverter() VisibilityQueryConverter
```

Comment:
Why do we have this Visibility prefix here?

Reply:
Just to be explicit that this query converter inside the `Plugin` is for Visibility. The `Plugin` interface is used to represent the whole DB implementation for both the core and Visibility stores.

Reply:
Discussed offline